cmd/srcd/daemon/daemon.go

```diff
@@ -74,8 +77,16 @@ func CleanUp() error {
 	}
 
 	gitbaseIndexDir := filepath.Join(datadir, "gitbase")
+	if err := os.RemoveAll(gitbaseIndexDir); err != nil {
+		return err
+	}
+
+	stateFile := filepath.Join(datadir, stateFileName)
+	if err := os.RemoveAll(stateFile); err != nil {
```

Comment:
if it's used, the rest is not needed.
```suggestion
	return os.RemoveAll(stateFile);
```